signer/src/stacks/contracts.rs

```diff
@@ -1074,27 +1095,38 @@ impl AsContractCall for RejectWithdrawalV1 {
     /// Validates that the reject-withdrawal-request satisfies the
     /// following criteria:
     ///
-    /// 1. That the transaction with the associated request_id is stored as
-    ///    an event on the canonical Stacks blockchain.
-    /// 2. That the signer bitmap matches the signer decisions stored in
-    ///    this signer's database.
+    /// 1. That the withdrawal request is not already completed.
+    /// 2. Whether the smart contract deployer matches the deployer in our
+    ///    context.
+    /// 2. Whether the associated withdrawal request transaction is
```

Comment:
Looks like this should be `3.`

Reply:
Ooops, will update.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1396/commits/e9eb8480e52b24ea9024e6df652c43193c80b312